core/src/com/fallenflame/game/LevelController.java

```diff
@@ -1050,7 +1053,7 @@ public void beginContact(Contact contact) {
             // Check for loss condition 1 (player runs into enemy)
             if((bd1 == player && bd2 instanceof EnemyModel)
                     || (bd1 instanceof  EnemyModel && bd2 == player)){
-                setLevelState(LevelState.LOSS);
+                player.die();
```

Comment:
It'll probably be worthwhile to, instead of doing this, send up a state (maybe LevelState.LOSING) that, if true, doesn't let the player move. It does feel really strange that I can sprint my character away as it falls to the floor. Either that or the animation needs to be faster, but I think stopping all movement would feel better.

Reply:
So, I made a one line fix that essentially stops the player from moving. When the player is hit by any enemy they now cannot move, however, the player can still be pushed by the walking enemy a very short distance (maybe at most 1m). If any other enemy hits the player they stop on a dime. I think it's ok rn.